### DISCUSSION TITLE
Theme Editor

### DISCUSSION THREAD

**Discussion by johnno1956** [LibreSprite/LibreSprite#388] [Q&A]
I have downloaded the several themes available for LibreSprite and as I was reading through them I was told that they were made by the community. I googled and searched for a theme editor but could not find one.

I would like to know how the community creates a theme/skin. Do they use a dedicated theme/skin editor or the laborious task of modifying the sheet.png, font/minifont.png and the skin.xml files?

**Comment by maujin111** [comment]
There is not a dedicated software to modify the themes/skin, the themes/skins are made modifying sheet.png, font/minifont.png and the skin.xml files. To modify the sheet.png you can use Libresprite and for the skin.xml file you can use any text editor. If you need any help there is the discord server where you can find help.

**Reply by johnno1956** [reply]
Thank you. I thought that changing the files might be the way to go... Discord server... Ok. I will pop over there and "bend their ear"...  Much appreciated.

**Comment by jjhaggar** [comment]
If you are on Windows, I'd recommend "Notepad++" as text editor. You better don't use regular "Notepad", since it has only one "undo" and you could lose all your work in a second.

**Reply by johnno1956** [reply]
Thank you for the tip... I will keep that in mind...
